core/tests/test_utils.py

```diff
@@ -74,11 +74,30 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Set the content, status_code and headers to return in subsequent
+        calls to the urlfetch mock.
+
+        Args:
+            content: str. The content to return in subsequent call to the
+            urlfetch mock.
+            status_code: int. The status_code to return in subsequent calls to
+            the urlfetch mock.
+            headers: dict. The headers to return in subsequent calls to the
+            urlfetch mock. The keys of this dict are strings that represent
+            the header name and the value represents corresponding value of
```

Comment:
... and each value represents the corresponding ...

Reply:
Done!